enterprise/frontend/src/metabase-enterprise/documents/components/EmojiPicker/EmojiPicker.tsx

```diff
@@ -0,0 +1,93 @@
+import {
+  type EmojiPickerListCategoryHeaderProps,
+  type EmojiPickerListEmojiProps,
+  type Emoji as EmojiType,
+  EmojiPicker as Picker,
+} from "frimousse";
+import { type ChangeEvent, useState } from "react";
+import { t } from "ttag";
+
+import { ActionIcon, Box, Icon, Text, TextInput } from "metabase/ui";
+
+import S from "./EmojiPicker.module.css";
+
+// Snapshot from https://cdn.jsdelivr.net/npm/emojibase-data@16.0.3/en/data.json
+const EMOJIBASE_URL = "/app/assets/emoji";
```

Comment:
let's avoid adding this file and fetch it directly instead. we can move it to our backend later.